site_2/index.html

```diff
@@ -8,17 +8,40 @@
     <meta lang="en">
     <link rel="stylesheet" href="css/main.css">
     <link rel="stylesheet" href="scss/main.scss">
-
-    <!-- <link rel="preconnect" href="https://fonts.gstatic.com"> -->
-    <!-- <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;500&display=swap" rel="stylesheet">
-    <link rel="stylesheet" href="//cdn.materialdesignicons.com/5.4.55/css/materialdesignicons.min.css"> -->
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Mukta:wght@300;400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="//cdn.materialdesignicons.com/5.4.55/css/materialdesignicons.min.css">
 </head>
 
 <body>
     <div class="wrapper">
-        <header class="header">
-            <div class="logo"></div>
+        <header class="header header__box">
+            <div class="header__logo">kh.</div>
+
+            <div class="header__item header__icon-search ">
```

Comment:
Кажется два элемента тоже не могут быть на одном теге, нужен модификатор

```suggestion
            <div class="header__item header__item--icon-search ">
```